docs/sdk/php.md

```diff
@@ -0,0 +1,90 @@
+## PHP SDK
+
+### Installation
+
+Installation is best done using [composer](https://getcomposer.org/)
+
+Create a composer.json file with the following:
+
+```json
+{
+    "repositories": {
+        "tulip/hydra-php-sdk": { "type": "vcs", "url": "git@git.internal.tulip.io:shared/composer-libraries/hydra-php-sdk.git" }
```

Comment:
I guess this can be updated? I registered packagist here: https://packagist.org/packages/ory/hydra-sdk

Reply:
Oops, the above was from internal testing. Fixed this reference to your packagist repo.